context/app/static/js/stores/useSavedEntitiesStore.js

```diff
@@ -65,6 +65,11 @@ const useSavedEntitiesStore = create(
           state.listsToBeDeleted = [];
         });
       },
+      deleteList: (listTitle) => {
+        set((state) => {
+          delete state.savedLists[listTitle];
```

Comment:
Am I understanding it correctly, that one of the big selling points of immer is that we can use idioms like this that mutate the object, instead of making work-arounds?

Reply:
Yep!